plugins/inputs/directory_monitor/README.md

```diff
@@ -39,6 +39,10 @@ This plugin is intended to read files that are moved or copied to the monitored
   ## Lowering this value will result in *slightly* less memory use, with a potential sacrifice in speed efficiency, if absolutely necessary.
   #	file_queue_size = 100000
   #
+  ## Name a tag containing the name of the file the data was parsed from.  Leave empty
+  ## to disable.
+  # file_tag = ""
+  #
```

Comment:
I have two concerns:

1) Creating a tag for every file has the potential of increasing the [cardnality](https://docs.influxdata.com/influxdb/cloud/reference/glossary/#series-cardinality) greatly. At the very least, I think this needs to be pointed out in this comment.

2) Is there a reason why this tag name is configurable and not just "filename"?

Reply:
Hi Joshua,

Thanks for having a look at my PR. 
Regarding your first comment, I have to say that this is not a new feature added to Telegraf.
I have got my inspiration from https://github.com/influxdata/telegraf/pull/6650. I do however acknowledge that it would be good to have an option to add the file name as fields, and not just tags. I will add that to a different feature branch.
And finally, the Tag Name is desirable to be configurable. For example, if the filename determines the type of device the information was generated from, then you would like to have the tag be called "device_type".

I can add "Cautious when file name variation is high, this can increase the cardinality significantly."

Could you also let me know how I can get the "CLA Signed" to pass? I have already filled the form and submitted it. Thanks.

Kind regards,
Ehsan

Reply:
Thanks for pointing out that other PR. I did find it odd, but your use-case makes sense. If you could still add the comment about cardinality that would be good.

You have a few tests failing here is what you need to do:
1) rebase on master to get the latest test code, which should get test-go-windows to pass
2) update your commit message to start with "feat:" as this is a new feature.
3) for the CLA I think I just re-kicked it, but it may also take a day or so.

Reply:
Hey Joshua,

Could you also have a look at this PR too? https://github.com/influxdata/telegraf/pull/9865
It is related to the same Directory monitor input plugin, thanks.

Also, how long does the review process take before PRs get into master?

Kind regards,
Ehsan